CONTRIBUTING.md

```diff
@@ -23,14 +23,14 @@ We apply the monorepo architecture; see each workspace's `README.md` for details
 Use the npm command's option `--workspace` with a workspace name to execute workspace-specific npm scripts, like:
 
 ```
-$ npm run develop --workspace=@viron/app
-$ npm run build --workspace=@viron/app --workspace=@viron/lib
+npm run dev --workspace=@viron/app
```

Comment:
As part of this update, the dev script has been standardized as the default local development command for each package within the monorepo.